community/kernel/src/main/java/org/neo4j/kernel/api/proc/Neo4jTypes.java

```diff
@@ -46,7 +46,7 @@
 
         public AnyType()
         {
-            this( "Any" );
+            this( "ANY?" );
```

Comment:
It's from the cypher type system CIP, we wanted it to be consistent - `?` means the type is nullable.